diff --git a/tools/workspace/fcl/fcl-create-cps.py b/tools/workspace/fcl/fcl-create-cps.py
index 08e5365..89f6432 100644
--- a/tools/workspace/fcl/fcl-create-cps.py
+++ b/tools/workspace/fcl/fcl-create-cps.py
@@ -35,7 +35,6 @@ content = """
       "Type": "dylib",
       "Location": "@prefix@/lib/libfcl.so",
       "Includes": [ "@prefix@/include" ],
-      "Requires": [ "ccd:ccd", "Eigen3:Eigen", "octomap:octomap" ]
     }
   }
 }
